### ISSUE TITLE
disable {$} autocompletion inside JS

### ISSUE THREAD

**Issue by gherkins** [raket/idea-silverstripe#48]
would there be a way to disable the {$} autocompletion inside `<script>` blocks?
Sometimes you'll have inline JS inside .ss templates. And it kind of gets in the way when typing something like:
$(function(){})

**Comment by MarioHofer** [comment]
+1 from my side

**Comment by Firesphere** [comment]
I agree with Zauber, because {} is far from often a reason to insert the variable.

**Comment by MarcusDalgren** [comment]
Ok I'll simply turn this feature off for now and do a quick release.

**Comment by MarioHofer** [comment]
Perfect for me. Thank you very much!

--
*knsqnt interactive*
Frankenstraße 3
92339 Beilngries

+49 176 96987451
email@knsqnt.com

www.knsqnt.com

**Comment by adrhumphreys** [comment]
Why was this added back? :C

**Comment by MarcusDalgren** [comment]
I didn't add it back.
There never was a release fixing this AFAIK.

**Comment by patricknelson** [comment]
Actually I'd love to be able to disable the insertion here all together. For example, I could be in a `.ss` template and writing up some mustache variables and it becomes totally unusable 😢 

![2017-04-27_14-54-59](https://cloud.githubusercontent.com/assets/4269377/25506177/3d481ab0-2b5a-11e7-8f64-45aae972f68a.gif)

> Ok I'll simply turn this feature off for now and do a quick release.

@MarcusDalgren Sorry to disturb as I know you've expressed your [lack of time to help maintain this project](https://groups.google.com/d/msg/silverstripe-dev/bozY5hh9g4c/CJPpXJOOBwAJ), hopefully this question will be quick. 

How can we disable this feature? e.g. Is it possible to setup a configuration screen somewhere in the `Settings` panel of PhpStorm where we can enable/disable? Or at least where in the code of this repo does this feature exist so we could possibly disable and re-build the plugin ourselves?

**Comment by MarcusDalgren** [comment]
@patricknelson  I'm checking this right now...

**Comment by MarcusDalgren** [comment]
@patricknelson Can you try out the new version?

**Comment by patricknelson** [comment]
Huzzah! Instant gratification, thanks @MarcusDalgren, it works for me 👍😄 

Works well with regular curly braces and also with `{$var}` variables wrapped in braces as well. Here's a gif because gifs are fun.

![2017-04-27_15-34-46](https://cloud.githubusercontent.com/assets/4269377/25507139/2dec2fb6-2b5f-11e7-99ba-d8b7ba46d408.gif)

**Comment by MarcusDalgren** [comment]
Awesome! 😄 
I'll leave the issue open for awhile in case any issues crop up.

**Comment by patricknelson** [comment]
@MarcusDalgren Ever since I updated, I found that when it completes a SilverStripe tag like `<% loop $Variable %><% end_loop %>` (with the cursor right in between the tags) I cannot press enter at all unless I press space or move my cursor out, only then can I create a new line.

![2017-04-28_14-17-36](https://cloud.githubusercontent.com/assets/4269377/25547690/79d0bb40-2c1d-11e7-9c95-03af8cbf41cd.gif)

**Note:** The second `<% loop ... %>` I created is a separate bug, but I figured I'd show my regular use case (which I have to frequently undo when I accidentally accept the suggestion). That bug has been around for a very long time.

**Comment by MarcusDalgren** [comment]
@patricknelson Bug confirmed.
However it should be unrelated. If I turn off the enter handler it goes back to normal behavior.
The plugin tries to place the caret properly when pressing enter but that seems to have changed now since now you get stuck. I'll check on why it suddenly adds an extra <% as well.

I don't know if the behavior changed when I compiled against the latest PHPStorm. The feature I turned off is so minor so these bugs can't be related to turning that off.
